js/src/Components/EntityForm.js

```diff
@@ -40,8 +40,8 @@ export const EntityForm = ({entityType, bundle, entity, onFieldChange, widgetCon
 
   return (
     <div className="item-form">
-      {Object.keys(formFields).map(fieldName => {
-        const field = formFields[fieldName];
+      {Object.keys(formFields.form).map(fieldName => {
```

Comment:
I am getting a `typeError: Cannot convert undefined or null to object` here.

Step to reproduce.
1. Fresh build of 1.x cardinalsites
2. Check out this branch
3. Copy the example behavior file to stanford_basic
4. Enable the two example options on a paragraph row (admin/structure/paragraphs_row_type/node_stanford_page_row/edit)
5. Go and edit an existing page node.
6. Use the kebab menu to edit the row and see error

Additional error of 
```
Warning: uasort() expects parameter 1 to be array, null given in Drupal\react_paragraphs\Plugin\rest\resource\ReactParagraphsResource->get() (line 144 of modules/custom/react_paragraphs/src/Plugin/rest/resource/ReactParagraphsResource.php).
```

Reply:
Should be fixed now